src/plugins/data_table/data_table.js

```diff
@@ -12,7 +12,8 @@ var LocalDataTable = (function() {
       this.options = options || {};
       // copy over certain properties from options to the table itself
       _.extend(this, _.pick(this.options, [ "selectedIds" ]));
-      _.bindAll(this, "_onRowCreated", "_onBulkHeaderClick", "_onBulkRowClick", "_bulkCheckboxAdjust", "_onDraw", "_onColumnVisibilityChange");
+      _.bindAll(this, "_onRowCreated", "_onBulkHeaderClick", "_onBulkRowClick", "_bulkCheckboxAdjust", "_onDraw",
+          "_onColumnVisibilityChange", "_onReorder");
```

Comment:
Is '_onReorder' too generic of a name? Colin said there is a separate function in the reordering plugin with a similar name. We came with '_onBdColumnReorder' as a possible other function name.